qemu/tests/block_copy.py

```diff
@@ -143,9 +143,11 @@ def is_paused(self):
         """
         paused = self.get_status().get("paused")
         offset_p = self.get_status().get("offset")
-        time.sleep(random.uniform(1, 3))
-        offset_l = self.get_status().get("offset")
-        paused &= offset_p == offset_l
+        paused &= bool(utils_misc.wait_for(lambda: not self.get_status().get("busy"),
+                                           timeout=5))
+        if paused:
+            offset_l = self.get_status().get("offset")
+            paused &= offset_p == offset_l
```

Comment:
@vivianQizhu 
I think this modification can not fix the problem.
During my test,  after get ("paused") is "True",  the value of "offset" still can change if "busy" is "True".   So if get  "offset_p" before "busy" turn "False",   offset_p will not equal to offset_1, and return paused failed.
I think it should get "offset_p" after "busy" turn "False".

if paused:
    offset_p = self.get_status().get("offset")
    time.sleep(random.uniform(1, 3))
    offset_l = self.get_status().get("offset")
    paused &= offset_p == offset_l